### PR TITLE
USHIFT-1300: Avoid getting cluster state for TEST_PROVIDER env var

### PR DESCRIPTION
**Pull request by pacevedom**
/cc @soltysh 
/cc @ingvagabund

### CODE REVIEW

File: pkg/test/ginkgo/test_runner.go [hunk 1]
```diff
@@ -362,9 +362,12 @@ func updateEnvVars(envs []string) []string {
 	}
 	// copied from provider.go
 	// TODO: add error handling, and maybe turn this into sharable helper?
+	config := &exutilcluster.ClusterConfiguration{}
 	clientConfig, _ := framework.LoadConfig(true)
 	clusterState, _ := exutilcluster.DiscoverClusterState(clientConfig)
```
Comment (ingvagabund) [inline pkg/test/ginkgo/test_runner.go:367]: What if the error return by `exutilcluster.DiscoverClusterState` is valid?

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: pkg/test/ginkgo/test_runner.go (hunk 1 above)
Comment (soltysh) [reply pkg/test/ginkgo/test_runner.go:367]: Yeah, we need to add proper error handling :sweat_smile: just like that TODO says

**Pull request merged**